JS/main.js

```diff
@@ -1,12 +1,5 @@
 let myBooks = [];
 
-class Book {
-  constructor(title, author) {
-    this.title = title;
-    this.author = author;
-  }
-}
-
 function getBooks() {
   myBooks = JSON.parse(localStorage.myBooks);
   const books = document.getElementById('books');
```

Comment:
- [ ] All of the functionality of this project should be embedded inside classes, please move this function in your class